src/Schema/Item.php

```diff
@@ -6,61 +6,38 @@
 
 abstract class Item implements SchemaItem
 {
-    protected $doc;
-
-    protected $schema;
-
-    protected $name;
+    use NamedItemTrait;
+    use SchemaItemTrait;
 
+    /**
+     * @var Type|null
+     */
     protected $type;
 
+    /**
+     * @param string $name
```

Comment:
missing `Schema $schema`

Reply:
without any additional summary information, the docblock is on the superfluous side of things- on the php-7.1 branch docblocks like these are entirely removed as the type information for all parameters is a native part of the declaration.